cdc/model/sink.go

```diff
@@ -269,6 +270,15 @@ type RowChangedEvent struct {
 
 	// SplitTxn marks this RowChangedEvent as the first line of a new txn.
 	SplitTxn bool `json:"-" msg:"-"`
+	// This filed is only used by expression filter.
+	// It will be set to nil after used.
+	RowChangedDatums *RowChangedDatums `json:"-" msg:"-"`
```

Comment:
```suggestion
	RowChangedDatums RowChangedDatums `json:"-" msg:"-"`
```

Prefer value to ease GC pressure.

Reply:
We will assign a value to this filed and set it to nil in a same function scope, so I think it is acceptable use a pointer here.